cmd/di.go

```diff
@@ -394,6 +396,8 @@ func (di *Dependencies) bootstrapStorage(path string) error {
 
 	di.Storage = localStorage
 
+	netutil.SetRouteManagerStorage(di.Storage)
```

Comment:
Why not to have this in factory? And call ClearStaleRoutes() on routeManager instance?

Reply:
Routes cleaning is a system-wide component. You never need to have more than one of them. Having it independent allows to access it simpler, without passing it down through all components.